src/core/api/public_api.ts

```diff
@@ -1363,9 +1366,37 @@ class Player extends EventEmitter<IPublicAPIEvent> {
   /**
    * Update the playback rate of the video.
    * @param {Number} rate
+   * @param {Boolean|undefined} enableTrickModeTrack
    */
-  setPlaybackRate(rate : number) : void {
+  setPlaybackRate(rate : number, enableTrickModeTrack?: boolean) : void {
     this._priv_speed$.next(rate);
+    if (enableTrickModeTrack !== undefined) {
```

Comment:
May be more readable by writing:

```js
if (enableTrickModeTrack  === true) {
  // ...
} else if (enableTrickModeTrack === false) {
  // ...
```

Or even by returning early when not a bool:
```js
if (typeof enableTrickModeTrack !== "boolean") {
  return;
}
if (enableTrickModeTrack) {
  // ...
} else {
  // ...
```

Reply:
done